### ISSUE TITLE
WebGL build

### ISSUE THREAD

**Issue by AmirArdroudi** [Nethereum/Unity3dSimpleSampleNet461#6]
Hey guys, I have some errors on building for webgl. any idea?

`ArgumentException: The Assembly WindowsBase is referenced by System.Reactive ('Assets/Plugins/System.Reactive.dll'). But the dll is not allowed to be included or could not be found.
UnityEditor.AssemblyHelper.AddReferencedAssembliesRecurse (System.String assemblyPath, System.Collections.Generic.List`1[T] alreadyFoundAssemblies, System.String[] allAssemblyPaths, System.String[] foldersToSearch, System.Collections.Generic.Dictionary`2[TKey,TValue] cache, UnityEditor.BuildTarget target) (at <7587a42c8322471cbc42b27c9b8eab3c>:0)
UnityEditor.AssemblyHelper.AddReferencedAssembliesRecurse (System.String assemblyPath, System.Collections.Generic.List`1[T] alreadyFoundAssemblies, System.String[] allAssemblyPaths, System.String[] foldersToSearch, System.Collections.Generic.Dictionary`2[TKey,TValue] cache, UnityEditor.BuildTarget target) (at <7587a42c8322471cbc42b27c9b8eab3c>:0)
UnityEditor.AssemblyHelper.FindAssembliesReferencedBy (System.String[] paths, System.String[] foldersToSearch, UnityEditor.BuildTarget target) (at <7587a42c8322471cbc42b27c9b8eab3c>:0)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr, Boolean&)`

**Comment by StinkySteak** [comment]
is system.reactive included on nethereum? try to use Unity .NET dll instead

**Comment by juanfranblanco** [comment]
I have included all dlls so everyone can experiment at their own will. I might create different versions of the packaged dlls (advance mode with all).

**Comment by AmirArdroudi** [comment]
> I have included all dlls so everyone can experiment at their own will. I might create different versions of the packaged dlls (advance mode with all).

I've just clone [this repo](https://github.com/Nethereum/Unity3dSimpleSampleNet461/) and wanted to build it for WebGL without any changes in dlls. And yeah, system.reactive is included in the repo. 
I've also tried to build it with .NET 4 in player settings, but I get the same error every time.

**Comment by juanfranblanco** [comment]
@AmirArdroudi it is my bad when you are working on webgl you won't need half of the dlls, I will make a webgl specific version.

**Comment by Hotmanics** [comment]
@juanfranblanco Hey, still seeing the issue. However It sounds like simply disabling some DLLs will allow it to work properly.

![Capture](https://user-images.githubusercontent.com/32080359/131364163-980d83bb-1130-4b8f-944e-96cc4c7c431f.PNG)

You wont need to create a new repo or a webgl specific version. You can simply toggle these options for the build versions you would like.

**Comment by juanfranblanco** [comment]
Yes, the DLLs are added for completeness. So just a minimal set for users getting started as they don't know which ones might be ideal.

**Comment by akshay310599** [comment]
ibrary\Bee\artifacts\WebGL\build\debug_WebGL_wasm\build.js: undefined symbol: sendfile 

This is the issue i am receiving when  building for WebGL, what do i need to do. I just cloned and build the exact project.

**Comment by juanfranblanco** [comment]
Library\Bee ?

**Comment by juanfranblanco** [comment]
There is another template https://github.com/Nethereum/Nethereum.Unity.Webgl

**Comment by juanfranblanco** [comment]
on library bee https://forum.unity.com/threads/library-bee-error-when-trying-to-build-for-webgl.1220706/

**Comment by juanfranblanco** [comment]
Glad you are sorted, what a pain the migration

**Comment by Yuriy-Ihor** [comment]
> There is another template https://github.com/Nethereum/Nethereum.Unity.Webgl

This is for Unity 2021+, but not for 2020. Is there any other working templates for 2020?

**Comment by chiuan** [comment]
for the "Library\Bee\artifacts\WebGL\build\debug_WebGL_wasm\build.js: undefined symbol: sendfile"

Did anyone found anything?
